src/AudioUnit/AudioUnit.cs

```diff
@@ -135,12 +135,21 @@ internal AudioUnitException (int k) : base (Lookup (k))
 	delegate AudioUnitStatus CallbackShared (IntPtr /* void* */ clientData, ref AudioUnitRenderActionFlags /* AudioUnitRenderActionFlags* */ actionFlags, ref AudioTimeStamp /* AudioTimeStamp* */ timeStamp, uint /* UInt32 */ busNumber, uint /* UInt32 */ numberFrames, IntPtr /* AudioBufferList* */ data);
 #endif // !COREBUILD
 
+#if NET
+	[StructLayout (LayoutKind.Sequential)]
+	unsafe struct AURenderCallbackStruct
+	{
+		public delegate* unmanaged<IntPtr, int*, AudioTimeStamp*, uint, uint, IntPtr, int> Proc;
```

Comment:
So the type problems here are due to our complicated two-step build process.

The first step (what we call the core build) is a smaller/simplified build, where we only include some of the files.
Between step 1 and 2 we use the output from the first step to generate all our generated bindings.
The second step is the final build where we produce the actual assemblies we ship, and include all our code + generated code.

The build errors occur because the types the compiler can't find aren't included in the first build.

The fix is to have different declarations depending on whether you're in the first build or second build:

```suggestion
#if COREBUILD
		public delegate* unmanaged<IntPtr, int*, AudioTimeStamp*, uint, uint, IntPtr, int> Proc;
#else
		public delegate* unmanaged<IntPtr, AudioUnitRenderActionFlags*, AudioTimeStamp*, uint, uint, IntPtr, AudioUnitStatus> Proc;
#endif
```

It looks like this will clean up a lot of new NET code in your PR.